data/presets/power/insulator.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon": "temaki-power",
+    "fields": [
+        "ref",
+        "operator",
+        "line_attachment"
+    ],
+    "moreFields": [
+        "height",
+        "line_arrangement",
```

Comment:
FWIW,  the combination `power=insulator` + `line_arrangement`  has (currently) zero uses in OSM: https://overpass-turbo.eu/s/1DGp

:shrug:

Reply:
Yes but it's a valid combination